images/jupyter-user/Dockerfile

```diff
@@ -144,7 +144,7 @@ RUN conda run pip install -r /opt/orbit/aws-orbit_jupyter-user/aws-orbit/require
 RUN conda run pip install /opt/orbit/aws-orbit_jupyter-user/aws-orbit
 RUN conda run pip install /opt/orbit/aws-orbit_jupyter-user/aws-orbit-sdk
 RUN conda run pip install /opt/orbit/aws-orbit_jupyter-user/jupyterlab_orbit
-RUN conda run pip install softwarelabs-remote-toolkit==0.2.0
+RUN conda run pip install softwarelabs-remote-toolkit~=0.1.0
```

Comment:
Noticed that we have pushed `0.1.1`version of `slrt`

Reply:
Yep...I can update